### PR TITLE
move to k8s.io v1.23.0 and controller-runtime v0.11.0-beta.0

### PR DESCRIPTION
**Pull request by mythi**
Signed-off-by: Mikko Ylinen <mikko.ylinen@intel.com>

### CODE REVIEW

File: pkg/fpgacontroller/patcher/patchermanager.go [hunk 1]
```diff
@@ -40,8 +40,8 @@ func init() {
 
 // Manager keeps track of patchers registered for different Kubernetes namespaces.
 type Manager struct {
-	log      logr.Logger
 	patchers map[string]*patcher
+	log      logr.Logger
```
Comment (bart0sh) [inline pkg/fpgacontroller/patcher/patchermanager.go:44]: where this change came from?

Comment (mythi) [reply pkg/fpgacontroller/patcher/patchermanager.go:44]: from `fieldalignment` linter check.

**Comment by mythi** [comment]
tested with the e2e `Admission` tests and `intel-deviceplugin-operator`.

**Review by bart0sh — APPROVED** [review]
/lgtm

**Review by rojkov — APPROVED** [review]

**Pull request merged**